main.go

```diff
@@ -65,7 +65,8 @@ func main() {
 		checkKeyGroups          = flag.String("check-key-groups", getEnv("REDIS_EXPORTER_CHECK_KEY_GROUPS", ""), "Comma separated list of lua regex for grouping keys")
 		checkStreams            = flag.String("check-streams", getEnv("REDIS_EXPORTER_CHECK_STREAMS", ""), "Comma separated list of stream-patterns to export info about streams, groups and consumers, searched for with SCAN")
 		checkSingleStreams      = flag.String("check-single-streams", getEnv("REDIS_EXPORTER_CHECK_SINGLE_STREAMS", ""), "Comma separated list of single streams to export info about streams, groups and consumers")
-		countKeys               = flag.String("count-keys", getEnv("REDIS_EXPORTER_COUNT_KEYS", ""), "Comma separated list of patterns to count, eg: 'db3=sessions:*'. Warning: The exporter runs SCAN to count the keys.")
+		countKeys               = flag.String("count-keys", getEnv("REDIS_EXPORTER_COUNT_KEYS", ""), "Comma separated list of patterns to count (eg: 'db0=production_*,db3=sessions:*'), searched for with SCAN")
+		checkKeysBatchSize      = flag.Int64("check-keys-batch-size", getEnvInt64("REDIS_EXPORTER_CHECK_KEYS_BATCH_SIZE", 1000), "Approximate number of keys to process in each execution, larger value speeds up scanning.\nWARNING: Still Redis is a single-threaded app, huge COUNT can affect production environment.")
```

Comment:
New application flag added.
I am interested in discussing adding a shared flag for `checkKeysBatchSize` and `checkKeyGroupsBatchSize`. It would be a breaking change, but we can set up a fallback rule for key-group to use the `checkKeysBatchSize` value if `checkKeyGroupsBatchSize` isn't specified.

Glad to hear @rmak-cpi as an author of `memory-usage metrics` functionality where the COUNT option was involved.

Reply:
I think one (shared) flag for both is fine.